cmd/api/main.go

```diff
@@ -96,6 +98,18 @@ func main() {
 		log.Fatal().Err(err).Msg("creating user store")
 	}
 
+	metricsDatabaseURL := fmt.Sprintf(
```

Comment:
Decided to go with another database called `metrics.db` to keep things decoupled. Metrics are not part of the protocol and are still an experimental thing.